test/unit/sorted-views.spec.js

```diff
@@ -391,4 +391,30 @@ describe('collection/composite view sorting', function() {
       });
     });
   });
+
+  describe('when using `{ reorderOnSort: true }`', function () {
+    beforeEach(function () {
+      this.collectionView = new this.CollectionView({
+        childView: this.ChildView,
+        collection: this.collection,
+        reorderOnSort: true
```

Comment:
testing that `reorder` is called when `reorderOnSort` is set to true on the prototype would be good too.